tests/models/llava/test_modeling_llava.py

```diff
@@ -307,10 +307,50 @@ def test_small_model_integration_test_llama_batched_regression(self):
 
         output = model.generate(**inputs, max_new_tokens=20)
 
-        EXPECTED_DECODED_TEXT = ['USER:  \nWhat are the things I should be cautious about when I visit this place? What should I bring with me?\nASSISTANT: When visiting this serene location, one should be cautious about the weather conditions and potential', 'USER:  \nWhat is this?\nASSISTANT: Two cats lying on a bed!\nUSER:  \nAnd this?\nASSISTANT: A cat sleeping on a bed.']  # fmt: skip
+        EXPECTED_DECODED_TEXT = ['USER:  \nWhat are the things I should be cautious about when I visit this place? What should I bring with me?\nASSISTANT: When visiting this place, which appears to be a dock or pier extending over a body of water', 'USER:  \nWhat is this?\nASSISTANT: Two cats lying on a bed!\nUSER:  \nAnd this?\nASSISTANT: A cat sleeping on a bed.']  # fmt: skip
```

Comment:
This test itself was wrong. The input_ids is `tensor([[32001, 32001, 32001,     1,  3148,  1001, 29901, 29871, 32000, 29871,
            13,  5618,   526,   278,  2712,   306,   881,   367,   274,  1300,
          2738,  1048,   746,   306,  6493,   445,  2058, 29973,  1724,   881,
           306,  6963,   411,   592, 29973,    13, 22933,  9047, 13566, 29901],
        [    1,  3148,  1001, 29901, 29871, 32000, 29871,    13,  5618,   338,
           445, 29973,    13, 22933,  9047, 13566, 29901,  7803,   274,  1446,
         19214,   373,   263,  6592, 29991,    13, 11889, 29901, 29871, 32000,
         29871,    13,  2855,   445, 29973,    13, 22933,  9047, 13566, 29901]],
       device='cuda:0')`, with `32000` being the image tokens. The first sequences visibly see only the first image, so the output should not be different compared to https://github.com/huggingface/transformers/blob/00c1d87a7d5c8dfb4554370983b5a3f7c069edd7/tests/models/llava/test_modeling_llava.py#L245-L265, should it @younesbelkada?